### DISCUSSION TITLE
No syntax highlighting / preview window after plugin install (+restart)

### DISCUSSION THREAD

**Discussion by Capenus** [timo-reymann/intellij-mjml-support#82] [Q&A] [answered]
**OS:** Windows 10
**PhpStorm:** 2021.2.2, Build #PS-212.5284.49, built on September 16, 2021
**Non-Bundled Plugins:** net.seesharpsoft.intellij.plugins.csv (2.17.1), mobi.hsz.idea.gitignore (4.2.0), de.timo_reymann.intellij-mjml-support (1.19.0), de.espend.idea.php.annotation (8.0.0)
**Node:** v16.10.0
**npm:** 7.24.0

Hello,

I hope this is the right place to ask for help for this plugin.

I have reinstalled the plugin several times yesterday and also restarted phpstorm afterwards again and again. But I get no syntax highlighting and also the preview window does not open. Is there anything else I need to set or do? My files all have the extension .mjml or .html.mjml.

The only sign of life I get from the plugin is the message "copy renderer files for mjml preview" which is displayed at the bottom right near the event log after starting phpstorm.

I have attached two screenshots:

![grafik](https://user-images.githubusercontent.com/1089944/136156704-c99cf4df-b2f7-4af1-9582-7471993ba0a1.png)
![grafik](https://user-images.githubusercontent.com/1089944/136156799-d8bf32da-f416-4be3-a30f-842b5230a30d.png)

**Comment by timo-reymann** [comment, marked as answer]
Could you try to manually set it to MJML file type? Maybe it got associated with txt file type by accident.

To do so, right click and than:
![image](https://user-images.githubusercontent.com/21024948/136185714-8d3fe198-9dd8-46cb-872c-39a6f512b1fa.png)

You can also check (and override this in the IDE setttings):
![image](https://user-images.githubusercontent.com/21024948/136185917-d5a69cb8-fc00-4509-8adc-15947cb290d3.png)

**Reply by Capenus** [reply]
Thank you! That works. I had to delete .mjml from the File Type Text and add it back to MailJet. It was probably swapped when I was still working with mjml without the plugin.
